src/Storefront/Page/Robots/RobotsPageLoadedEvent.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Storefront\Page\Robots;
+
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\Event\NestedEvent;
+use Shopware\Core\Framework\Log\Package;
+use Symfony\Component\HttpFoundation\Request;
+
+#[Package('storefront')]
+class RobotsPageLoadedEvent extends NestedEvent
```

Comment:
```suggestion
class RobotsPageLoadedEvent extends NestedEvent implements ShopwareEvent
```

might be useful, as the event already has the context